test/unit/client/bundler/default.test.js

```diff
@@ -105,12 +110,19 @@ describe('default bundler:', function () {
         '", function (require, module, exports, __dirname, __filename){\n' + code + '\n});');
     });
 
+    it('should recognise existing asset bundle', function() {
+      var bundler = ss.api.bundler.get({ client: 'abc' });
+      bundler.latestPackedId.should.be.equal('123456789');
+    });
+
   });
 
   describe('client with relative css+code+tmpl', function() {
 
     var client;
 
+    beforeEach(function(done){ fixtures.reset(done); });
```

Comment:
Doesn't need to be changed, but this could be shortened to <code>beforeEach(fixtures.reset);</code>.